action/flags.go

```diff
@@ -1,7 +1,7 @@
 package action
 
-// FlagHandler contains all flags from cmd
-type FlagHandler struct {
+// BaseHandler contains all flags from cmd
+type BaseHandler struct {
 	Bench                bool   `json:"bench"`
 	ExpectSize           int64  `json:"expect_size"`
```

Comment:
`ExpectSize` only used in `Copy`

Reply:
> `ExpectSize` only used in `Copy`

You mean split the flag `expect-size` and the real `ExpectSize` apart ? Only handle the flag here and work with `ExpectSize` in `Copy` ?

Reply:
What I think is that we put all flag-related variables in `BaseHandler`, it's more like business-based logic than code-struct-based.